core/src/test/java/com/amazon/opendistroforelasticsearch/sql/expression/operator/predicate/UnaryPredicateOperatorTest.java

```diff
@@ -79,4 +104,93 @@ public void is_not_null_predicate() {
     assertEquals(BOOLEAN, expression.type());
     assertEquals(LITERAL_FALSE, expression.valueOf(valueEnv()));
   }
+
+  @Test
+  public void test_if_null_predicate() {
+    Expression v1 = dsl.literal(100);
+    Expression v2 = dsl.literal(200);
+
+    FunctionExpression result = dsl.ifnull(v1, v2);
+    assertEquals(v1.valueOf(valueEnv()), result.valueOf(valueEnv()));
+
+    v1 = DSL.literal(ExprNullValue.of());
+    result = dsl.ifnull(v1, v2);
+    assertEquals(v2.valueOf(valueEnv()), result.valueOf(valueEnv()));
+
+    v1 = dsl.literal(100);
+    v2 = DSL.literal(ExprNullValue.of());
+    result = dsl.ifnull(v1, v2);
+    assertEquals(v1.valueOf(valueEnv()), result.valueOf(valueEnv()));
+
+    v1 = DSL.literal(ExprNullValue.of());
+    v2 = DSL.literal(ExprNullValue.of());
+    result = dsl.ifnull(v1, v2);
+    assertEquals(v2.valueOf(valueEnv()), result.valueOf(valueEnv()));
+  }
+
+  @Test
+  public void test_null_if_predicate() {
+    Expression v1 = dsl.literal(100);
+    Expression v2 = dsl.literal(200);
+    FunctionExpression result = dsl.nullif(v1, v2);
+    assertEquals(v1.valueOf(valueEnv()), result.valueOf(valueEnv()));
+
+    System.out.println("debug : " + result.valueOf(valueEnv()));
+
+    v1 = dsl.literal(100);
+    v2 = dsl.literal(100);
+
+    result = dsl.nullif(v1, v2);
```

Comment:
remove the above line; compare the result with expected result?

Reply:
got it, thx